### ISSUE TITLE
[PFW]: Pasting list elements with numbers removes some of the numbers

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor-dev#2183]
## Type of report

Bug

1. Create Word document with below content:
```
1.	Test 1.2.3.4.5
2.	Test 
```
### Expected result

```
1.	Test 1.2.3.4.5
2.	Test 
```

### Actual result

```
1.	Test 2.3.4.5
2.	Test 
```
Notice that `1` from `Test 1.2.3.4.5` is gone.

## Other details

* Browser: Any
* OS: Any
* CKEditor version: 4.6.0+
* Installed CKEditor plugins: pastefromword

**Comment by msamsel** [comment]
@jswiderski it seems like issue doesn't exists any longer since 4.11.2

Can you confirm and close ticket in such case?

**Comment by Comandeer** [comment]
Duplicate of #2690, which was fixed in 4.11.2.
